kotlinx.fuzz.test/src/test/kotlin/kotlinx/fuzz/test/SampleTarget.kt

```diff
@@ -0,0 +1,20 @@
+package kotlinx.fuzz.test
+
+import kotlinx.fuzz.KFuzzTest
+import kotlinx.fuzz.KFuzzer
+import kotlinx.fuzz.KFuzzerImpl
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+// TODO: kotlin.test requires a class with empty ctor, NOT an object. Will this be a problem?
+class SampleTarget {
+
+    @KFuzzTest
+    @Test
+    fun test() {
+        // for now, just checking that all dependencies resolve
+        val kfuzzer: KFuzzer = KFuzzerImpl(byteArrayOf(1))
```

Comment:
It seems like my mistake, I believe that `KFuzzerImpl` should be internal, but I may be wrong

Reply:
I agree, I only use it here for testing purposes before we fully made the plugin functional